extensions/geode-modules-session-internal/src/main/java/org/apache/geode/modules/session/internal/common/ClientServerSessionCache.java

```diff
@@ -125,6 +127,10 @@ private void createOrRetrieveRegion() {
     } else {
       LOG.debug("Retrieved session region: " + this.sessionRegion);
     }
+
+    if (sessionRegion.getAttributes().getDataPolicy() != DataPolicy.EMPTY) {
```

Comment:
This method may call createLocalSessionRegion().  Which it looks like this same interest registration code is duplicated into that method?

Reply:
Sure but a few lines up we call createLocalSessionRegion().  If that gets called, we end up registering interest in that method also.  Just wanted to know why we do that or if that was intended...

Reply:
createLocalSessionRegion() in common/ClientServerSessionCache does not register interest. 

However, I will add register interest code in createLocalSessionRegion() and only register interest if is user defined the cache region.